src/app/page.tsx

```diff
@@ -1,8 +1,16 @@
 import { DndWrapper } from '@/hoc/DndWrapper'
 import { TaskBoard } from './ui/TaskBoard'
 import { Header } from '@/components/layouts/Header'
+import { z } from 'zod'
+import InvalidToken from '@/components/invalidToken'
```

Comment:
Please use PascalCase while naming component folders
```suggestion
import { z } from 'zod'
import InvalidToken from '@/components/InvalidToken'
```